modules/howtos/pages/observability-metrics.adoc

```diff
@@ -22,15 +21,8 @@ Right now only the first category is implemented by the SDK, more are planned.
 
 
 == The Default AggregatingMeter
-[.status]#Developer Preview#
 
 The default implementation aggregates and logs request and response metrics.
-Since the feature is in developer preview, you need to manually enable it on the environment:
```

Comment:
I removed this because it mentions _developer preview_ and tells you to enable it through the SDK.

I guess this would be enabled by default if the user has Couchbase Server 7.0? So I figured you won't need to show an example of enabling this anymore. But let me know if we should keep this.

Reply:
I think you're right.